warehouse/forklift/legacy.py

```diff
@@ -51,6 +51,7 @@
 from sqlalchemy.exc import MultipleResultsFound, NoResultFound
 
 from warehouse.admin.flags import AdminFlagValue
+from warehouse.attestations.models import Attestation as AttestationModel
```

Comment:
Nitpick: similar to what we do with macaroons, I suggest:

```suggestion
from warehouse.attestations.models import Attestation as DatabaseAttestation
```

(since the other `Attestation` is also technically a model, just not a DB one.)